modular_skyrat/modules/mapping/code/space.dm

```diff
@@ -9,14 +9,14 @@
 	description = "Cybersun would like to remind it's employees that any battle cruiser will be apropriately maintained, as will it's crew."
 	suffix = "forgottenship_skyrat.dmm"
 	always_place = TRUE
-*/
+
```

Comment:
Instead of removing the comment please begin a new one

Reply:
And instead of commenting it out in the first place you can simply `unpickable = TRUE`, that allows admins to still load it in manually